<!doctype html><html lang=en><head><meta name=generator content="Hugo 0.77.0"><title>Athena Chen</title><meta name=description content="Portfolio and personal blog of Athena Chen."><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta http-equiv=x-ua-compatible content="ie=edge"><link rel=stylesheet href=/css/bootstrap.min.css><link rel=stylesheet href=/css/layouts/main.css><link rel=stylesheet href=/css/navigators/navbar.css><link rel=stylesheet href=/css/plyr.css><link rel=stylesheet href=/css/flag-icon.min.css><link rel=stylesheet href="https://fonts.googleapis.com/css2?family=Muli:wght@300;400;500;600"><link rel=stylesheet href=/fontawesome/css/all.min.css><link rel=icon type=image/png href=/images/site/logo_hu4d503d6cbea57a74da0f7fb86944780b_6724_42x0_resize_box_2.png><meta property="og:title" content="Athena Chen"><meta property="og:type" content="website"><meta property="og:description" content="Portfolio and personal blog of Athena Chen."><meta property="og:image" content="https://athchen.github.io/images/author/athchen.jpg"><meta property="og:url" content="https://athchen.github.io"><link rel=stylesheet href=/css/sections/home.css><link rel=stylesheet href=/css/sections/about.css><link rel=stylesheet href=/css/sections/skills.css><link rel=stylesheet href=/css/sections/experiences.css><link rel=stylesheet href=/css/sections/education.css><link rel=stylesheet href=/css/sections/projects.css><link rel=stylesheet href=/css/sections/recent-posts.css><link rel=stylesheet href=/css/sections/achievements.css><link rel=stylesheet href=/css/sections/accomplishments.css><link rel=stylesheet href=/css/style.css></head><body data-spy=scroll data-target=#top-navbar data-offset=100><nav class="navbar navbar-expand-xl top-navbar initial-navbar" id=top-navbar><div class=container><a class=navbar-brand href=/><img src=/images/site/logo_hu4d503d6cbea57a74da0f7fb86944780b_6724_42x0_resize_box_2.png id=logo alt=Logo>
Athena Chen</a>
<button class="navbar-toggler navbar-dark" id=navbar-toggler type=button data-toggle=collapse data-target=#top-nav-items aria-label=menu>
<span class=navbar-toggler-icon></span></button><div class="collapse navbar-collapse" id=top-nav-items><ul class="navbar-nav ml-auto"><li class=nav-item><a class=nav-link href=#home>Home</a></li><li class=nav-item><a class=nav-link href=#projects>Projects</a></li><li class=nav-item><a class=nav-link href=#publications>Publications</a></li><li class=nav-item><a class=nav-link href=#talks>Talks</a></li><li class=nav-item><a class=nav-link href=#teaching>Teaching</a></li><div class=dropdown-divider id=top-navbar-divider></div><li class=nav-item><a class=nav-link href=/files/cv.pdf>CV</a></li></ul></div></div><img src=/images/site/logo_hu4d503d6cbea57a74da0f7fb86944780b_6724_42x0_resize_box_2.png class=d-none id=main-logo alt=Logo>
<img src=/images/site/logo_hu4d503d6cbea57a74da0f7fb86944780b_6724_42x0_resize_box_2.png class=d-none id=inverted-logo alt="Inverted Logo"></nav><div class="container-fluid home" id=home><style>#homePageBackgroundImageDivStyled{background-image:url(https://athchen.github.io/images/site/blue_pink_bkg_hu707894bce38528afc44dd546e0023d69_63057_500x0_resize_box_2.png)}@media(min-width:500px) and (max-width:800px){#homePageBackgroundImageDivStyled{background-image:url(https://athchen.github.io/images/site/blue_pink_bkg_hu707894bce38528afc44dd546e0023d69_63057_800x0_resize_box_2.png)}}@media(min-width:801px) and (max-width:1200px){#homePageBackgroundImageDivStyled{background-image:url(https://athchen.github.io/images/site/blue_pink_bkg_hu707894bce38528afc44dd546e0023d69_63057_1200x0_resize_box_2.png)}}@media(min-width:1201px) and (max-width:1500px){#homePageBackgroundImageDivStyled{background-image:url(https://athchen.github.io/images/site/blue_pink_bkg_hu707894bce38528afc44dd546e0023d69_63057_1500x0_resize_box_2.png)}}@media(min-width:1501px){#homePageBackgroundImageDivStyled{background-image:url(https://athchen.github.io/images/site/blue_pink_bkg.png)}}</style><span class=on-the-fly-behavior></span><div id=homePageBackgroundImageDivStyled class="background container-fluid"></div><div class="container content text-center"><img src=/images/author/athchen_hu27f7dc34c9db8fd2b063f3fba6be9864_302162_148x148_fit_q75_box.jpg class="rounded-circle mx-auto d-block img-fluid" alt="Author Image"><h1 class=greeting>Athena Chen</h1><h5 class=greeting-subtitle>Data Scientist, Biostatistician</h5><p class=greeting>I recently received my PhD from the <a href=https://www.jhsph.edu/departments/biostatistics/>Department of Biostatistics</a> at the <a href=https://www.jhsph.edu/>Johns Hopkins Bloomberg School of Public Health</a>. For my dissertation work, I developed statistical methods to characterize antibody responses to various antigens using phage-immunoprecipitation sequencing technology (<a href=https://www.nature.com/articles/s41596-018-0025-6>PhIP-Seq</a>) with <a href=http://www.biostat.jhsph.edu/~iruczins/>Dr. Ingo Ruczinski</a> and <a href=http://labs.pathology.jhu.edu/larman/>Dr. H. Benjamin Larman</a>.
My research interests include Bayesian statistics, statistical genomics and proteomics, data analysis and data analysis teaching, and immunology.</p><ul class="social-link d-flex justify-content-center"><li><a href=mailto:achen70@jhu.edu title=Email target=_blank rel=noopener><i class="far fa-envelope"></i></a></li><li><a href=https://www.github.com/athchen title=Github target=_blank rel=noopener><i class="fab fa-github"></i></a></li><li><a href="https://scholar.google.com/citations?hl=en&user=BVuiP-kAAAAJ" title="Google Scholar" target=_blank rel=noopener><i class="fas fa-graduation-cap"></i></a></li><li><a href=https://www.linkedin.com/in/athena-chen-3706b6a6/ title=LinkedIn target=_blank rel=noopener><i class="fab fa-linkedin"></i></a></li></ul><a href=#projects aria-label="Read More - Athena Chen"><i class="arrow bounce fa fa-chevron-down"></i></a></div></div><div class="container-fluid section-holder d-flex bg-white"><div class="container-fluid anchor pb-5 projects-section" id=projects><h1 class=text-center>Projects</h1><div class="container ml-auto text-center"><div class="btn-group flex-wrap" role=group id=project-filter-buttons><button type=button class="btn btn-dark" data-filter=all>
All</button>
<button type=button class="btn btn-dark" data-filter=current>
Current</button>
<button type=button class="btn btn-dark" data-filter="R package">
R packages</button>
<button type=button class="btn btn-dark" data-filter="statistical method">
Statistical methods</button></div></div><div class="container filtr-projects"><div class=row id=project-card-holder><div class="col-sm-12 col-md-6 col-lg-4 p-2 filtr-item" data-category="all, current,R package,statistical method"><div class="card mt-1"><div class=card><a class=card-header href=https://github.com/athchen/beer target=_blank rel=noopener><div><div class=d-flex><img class=card-img-xs src=/images/hex/beer_phage_hu6b95ef6241de4d2ab49cc680737d4745_2828007_24x24_fit_box_2.png alt=BEER><h5 class="card-title mb-0">BEER</h5></div><div class=sub-title><span>Developer</span>
<span>Apr 2020 - Present</span></div></div></a><div class="card-body text-justify pt-1"><p>Bayesian method for identifying enriched antibody responses from PhIP-Seq data.</p><div class=project-card-footer><div class=project-tags-holder><span><a class="badge btn-info" href=https://www.biorxiv.org/content/10.1101/2022.01.19.476926v1 target=_blank><i class="far fa-file-alt pr-1 disabled" href></i>Pre-print</a></span>
<span><a class="badge btn-info" href=https://github.com/athchen/beer target=_blank><i class="fab fa-github pr-1 disabled" href></i>Github</a></span>
<span><a class="badge btn-info" href=https://bioconductor.org/packages/devel/bioc/html/beer.html target=_blank><i class="fas fa-music pr-1 disabled" href></i>Bioconductor</a></span></div></div></div></div></div></div><div class="col-sm-12 col-md-6 col-lg-4 p-2 filtr-item" data-category="all, current,R package,data infrastructure"><div class="card mt-1"><div class=card><a class=card-header href=https://bioconductor.org/packages/release/bioc/html/PhIPData.html target=_blank rel=noopener><div><div class=d-flex><h5 class="card-title mb-0">PhIPData</h5></div><div class=sub-title><span>Developer</span>
<span>Feb 2021 - Present</span></div></div></a><div class="card-body text-justify pt-1"><p>Package defining an S4 class for PhIP-Seq experiments. PhIPData allows users to coordinate metadata with experimental data in analyses.</p><div class=project-card-footer><div class=project-tags-holder><span><a class="badge btn-info" href=https://github.com/athchen/PhIPData target=_blank><i class="fab fa-github pr-1 disabled" href></i>Github</a></span>
<span><a class="badge btn-info" href=https://bioconductor.org/packages/release/bioc/html/PhIPData.html target=_blank><i class="fas fa-music pr-1 disabled" href></i>Bioconductor</a></span></div></div></div></div></div></div><div class="col-sm-12 col-md-6 col-lg-4 p-2 filtr-item" data-category="all, statistical method"><div class="card mt-1"><div class=card><a class=card-header href=https://onlinelibrary.wiley.com/doi/abs/10.1002/sim.8920 target=_blank rel=noopener><div><div class=d-flex><h5 class="card-title mb-0">kTSP classifier for HIV cross-sectional incidence estimation</h5></div><div class=sub-title><span>Research Assistant</span>
<span>May 2018 - Feb 2020</span></div></div></a><div class="card-body text-justify pt-1"><p>Extension of the kTSP classifier to longitudinal data to identify recent infections for HIV cross-sectional incidence estimation.</p><div class=project-card-footer><div class=project-tags-holder><span><a class="badge btn-info" href=https://github.com/athchen/ktsp_paper target=_blank><i class="fab fa-github pr-1 disabled" href></i>Github</a></span>
<span><a class="badge btn-info" href=https://onlinelibrary.wiley.com/doi/abs/10.1002/sim.8920 target=_blank><i class="far fa-file-alt pr-1 disabled" href></i>Publication</a></span></div></div></div></div></div></div><div class="col-sm-12 col-md-6 col-lg-4 p-2 filtr-item" data-category="all, current"><div class="card mt-1"><div class=card><a class=card-header href=javascript:void(0)><div><div class=d-flex><h5 class="card-title mb-0">Human phageome and toxome antigen library</h5></div><div class=sub-title><span>Research Assistant</span>
<span>Jun 2019 - Present</span></div></div></a><div class="card-body text-justify pt-1"><p>Annotated antigen library derived from metagenomic sequencing of stool samples and analyzed data with the new library.</p><div class=project-card-footer><div class=project-tags-holder><span><a class="badge btn-info" href=https://www.biorxiv.org/content/10.1101/2021.10.01.462481v1 target=_blank><i class="far fa-file-alt pr-1 disabled" href></i>Pre-print</a></span></div></div></div></div></div></div><div class="col-sm-12 col-md-6 col-lg-4 p-2 filtr-item" data-category="all, Computational Biology"><div class="card mt-1"><div class=card><a class=card-header href=https://www.molbiolcell.org/doi/10.1091/mbc.E20-08-0530 target=_blank rel=noopener><div><div class=d-flex><h5 class="card-title mb-0">Spatial Cell Modeling</h5></div><div class=sub-title><span>Research Assistant</span>
<span>Dec 2015 - Aug 2017</span></div></div></a><div class="card-body text-justify pt-1"><p>Analyzed and assessed challenges facing current single-particle modeling methods of biochemical systems.</p><div class=project-card-footer><div class=project-tags-holder><span><a class="badge btn-info" href=https://www.molbiolcell.org/doi/10.1091/mbc.E20-08-0530 target=_blank><i class="far fa-file-alt pr-1 disabled" href></i>Publication</a></span></div></div></div></div></div></div></div></div></div></div><div class="container-fluid section-holder d-flex bg-dimmed"><div class="container-fluid anchor pb-5 accomplishments-section" id=publications><h1 class=text-center>Featured Publications</h1><h6 class=text-center><em>A list of all publications can be found <a href="https://scholar.google.com/citations?hl=en&user=BVuiP-kAAAAJ">here</a></em></h6><div class=container><div id=acomplishment-card-holder><div><div class="col-md-12 col-lg-12 p-2"><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Detecting Antibody Reactivities in Phage ImmunoPrecipitation Sequencing Data</h5><div class=sub-title><span><strong>Chen, A.</strong>, Kammers, K., Larman, H. B., Scharpf R. B., Ruczinski, I.</span><br><span><a href=https://www.biorxiv.org/content/10.1101/2022.01.19.476926v1 title=bioRxiv target=_blank rel=noopener>bioRxiv</a></span>
<span class=ml-1>| Jan 2021</span></div></div><div class=card-body><p>Phage ImmunoPrecipitation Sequencing (PhIP-Seq) is a recently developed technology to assess antibody reactivity, quantifying antibody binding towards hundreds of thousands of candidate epitopes. The output from PhIP-Seq experiments are read count matrices, similar to RNA-Seq data; however some important differences do exist. In this manuscript we investigated whether the publicly available method edgeR for normalization and analysis of RNA-Seq data is also suitable for PhIP-Seq data. We find that edgeR is remarkably effective, but improvements can be made and introduce a Bayesian framework specifically tailored for data from PhIP-Seq experiments (Bayesian Enrichment Estimation in R, BEER).</p></div><div class=card-footer><span><a class="btn btn-outline-info ml-1 pl-2 mb-2" href target=_blank , rel=noopener role=button><i class="far fa-file-alt" style=color:#3b96cf></i>Pre-Print</a></span></div></div></div></div><div style="display:none;transition:all 1s ease-out"><div class="col-md-12 col-lg-12 p-2"><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Diagnosing Data Analytic Problems in the Classroom</h5><div class=sub-title><span>Peng, R., <strong>Chen, A.</strong>, Bridgeford, E., Leek, J. T. & Hicks, S. C.</span><br><span><a href=https://www.tandfonline.com/doi/full/10.1080/26939169.2021.1971586 title="Journal of Statistics and Data Science Education" target=_blank rel=noopener>Journal of Statistics and Data Science Education</a></span>
<span class=ml-1>| Aug 2021</span></div></div><div class=card-body><p>Teaching data analysis by providing students with real-world problems and datasets allows students to integrate a variety of skills in a situation that mirrors how data analysis actually works. However, whole data analyses may obscure the individual skills of data analytic practice that are generalizable across data analyses. One such skill is the ability to diagnose the cause of unexpected results in a data analysis. We describe an approach to teaching students skills in diagnosing data analytic problems.</p></div><div class=card-footer><span><a class="btn btn-outline-info ml-1 pl-2 mb-2" href target=_blank , rel=noopener role=button><i class="far fa-file-alt" style=color:#3b96cf></i>Publication</a></span></div></div></div></div><div style="display:none;transition:all 1s ease-out"><div class="col-md-12 col-lg-12 p-2"><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">HIV Antibody Profiles in HIV Controllers and Persons With Treatment-Induced Viral Suppression</h5><div class=sub-title><span>Kammers, K., <strong>Chen, A.</strong>, Monaco, D. R., Hudelson, S. E., Grant-McAuley, W., Moore, R. D., Alter, G., Deeks, S. G., Morrison, C. S., Eller, L. A., Blankson, J. N., Laeyendecker, O., Ruczinski, I., Eshleman, S. H. & Larman, H. B.</span><br><span><a href=https://www.frontiersin.org/articles/10.3389/fimmu.2021.740395/full title="Frontiers in Immunology" target=_blank rel=noopener>Frontiers in Immunology</a></span>
<span class=ml-1>| Aug 2021</span></div></div><div class=card-body><p>Low HIV viral load is associated with delayed disease progression and reduced HIV transmission. HIV controllers suppress viral load to low levels in the absence of antiretroviral treatment (ART). We used an antibody profiling system, VirScan, to compare antibody reactivity and specificity in HIV controllers, non-controllers with treatment-induced viral suppression, and viremic non-controllers.</p></div><div class=card-footer><span><a class="btn btn-outline-info ml-1 pl-2 mb-2" href target=_blank , rel=noopener role=button><i class="far fa-file-alt" style=color:#3b96cf></i>Pre-Print</a></span></div></div></div></div><div><div class="col-md-12 col-lg-12 p-2"><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">A top scoring pairs classifier for recent HIV infections</h5><div class=sub-title><span><strong>Chen, A.</strong>, Laeyendecker, O., Eshleman, S. H., Monaco, D. R., Kammers, K., Larman, H. B. & Ruczinski, I.</span><br><span><a href=https://onlinelibrary.wiley.com/doi/abs/10.1002/sim.8920 title="Statistics in Medicine" target=_blank rel=noopener>Statistics in Medicine</a></span>
<span class=ml-1>| Mar 2021</span></div></div><div class=card-body><p>As availability and use of antiretroviral treatment increase worldwide, algorithms that do not include HIV viral load and are not impacted by viral suppression are needed for cross-sectional HIV incidence estimation. Using a phage display system to quantify antibody binding to over 3300 HIV peptides, we present a classifier based on top scoring peptide pairs that identifies recent infections using HIV antibody responses alone.</p></div><div class=card-footer><span><a class="btn btn-outline-info ml-1 pl-2 mb-2" href target=_blank , rel=noopener role=button><i class="far fa-file-alt" style=color:#3b96cf></i>Publication</a></span>
<span><a class="btn btn-outline-info ml-1 pl-2 mb-2" href target=_blank , rel=noopener role=button><i class="fab fa-github" style=color:#3b96cf></i>Code</a></span></div></div></div></div><div style="display:none;transition:all 1s ease-out"><div class="col-md-12 col-lg-12 p-2"><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Antibody responses to endemic coronaviruses modulate COVID-19 convalescent plasma functionality</h5><div class=sub-title><span>Morgenlander, W. M., Henson, S. N., Monaco, D. R., <strong>Chen, A.</strong>, Littlefield, K., Bloch, E. M., Fujimura, E., Ruczinski, I., Crowley, A. R., Harini, N., Butler, S. E., Weiner, J. A., Li, M. Z., Bonny, T. S., Benner, S. E., Balagopal, A., Sullivan, D., Shoham, S., Quinn, T. C., Eshleman, S., Casadevall, A., Redd, A. D., Laeyendecker, O., Ackerman, M. E., Andrew, P., Elledge, S. J., Robinson, M. L., Tobian, A. A. R. & Larman, H. B.</span><br><span><a href=https://www.jci.org/articles/view/146927 title="The Journal of Clinical Investigation" target=_blank rel=noopener>The Journal of Clinical Investigation</a></span>
<span class=ml-1>| Feb 2021</span></div></div><div class=card-body><p>SARS-CoV-2 (CoV2) antibody therapies, including COVID-19 convalescent plasma (CCP), monoclonal antibodies, and hyperimmune globulin, are among the leading treatments for individuals with early COVID-19 infection. The functionality of convalescent plasma varies greatly, but the association of antibody epitope specificities with plasma functionality remains uncharacterized. We assessed antibody functionality and reactivities to peptides across the CoV2 and the 4 endemic human coronavirus (HCoV) genomes in 126 CCP donations.</p></div><div class=card-footer><span><a class="btn btn-outline-info ml-1 pl-2 mb-2" href target=_blank , rel=noopener role=button><i class="far fa-file-alt" style=color:#3b96cf></i>Publication</a></span></div></div></div></div><div style="display:none;transition:all 1s ease-out"><div class="col-md-12 col-lg-12 p-2"><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Quantifying the roles of space and stochasticity in computer simulations for cell biology and cellular biochemistry</h5><div class=sub-title><span>Johnson, M. E., <strong>Chen, A.</strong>, Faeder, J. R., Henning, P., Moraru, I. I., Meier-Schellersheim, M., Murphy, R. F., Prustel, T., Theriot, J. A. & Uhrmacher, A. M.</span><br><span><a href=https://www.molbiolcell.org/doi/10.1091/mbc.E20-08-0530 title="Molecular Biology of the Cell" target=_blank rel=noopener>Molecular Biology of the Cell</a></span>
<span class=ml-1>| Jan 2021</span></div></div><div class=card-body><p>Most of the fascinating phenomena studied in cell biology emerge from interactions among highly organized multimolecular structures embedded into complex and frequently dynamic cellular morphologies. For the exploration of such systems, computer simulation has proved to be an invaluable tool, and many researchers in this field have developed sophisticated computational models for application to specific cell biological questions. We have defined a series of computational test cases ranging from very simple to moderately complex, varying key features of dimensionality, reaction type, reaction speed, crowding, and cell size. We then quantified how explicit spatial and/or stochastic implementations alter outcomes, even when all methods use the same reaction network, rates, and concentrations.</p></div><div class=card-footer><span><a class="btn btn-outline-info ml-1 pl-2 mb-2" href target=_blank , rel=noopener role=button><i class="far fa-file-alt" style=color:#3b96cf></i>Publication</a></span></div></div></div></div><div style="display:none;transition:all 1s ease-out"><div class="col-md-12 col-lg-12 p-2"><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Comprehensive profiling of HIV antibody evolution</h5><div class=sub-title><span>Eshleman, S. H., Laeyendecker, O., Kammers, K., Chen, A., Sivay, M. V., Kottapalli, S., Sie, B. M., Yuan, T., Monaco, D. R., Mohan, D., Wansley, D., Kula, T., Morrison, C., Elledge, S. J., Brookmeyer, R., Ruczinski, I. & Larman, H. B.</span><br><span><a href=https://pubmed.ncbi.nlm.nih.gov/31042470/ title="Cell Reports" target=_blank rel=noopener>Cell Reports</a></span>
<span class=ml-1>| Apr 2019</span></div></div><div class=card-body><p>This study evaluates HIV antibody responses and their evolution during the course of HIV infection. A phage display system is used to characterize antibody binding to >3,300 HIV peptides in 57 adults with early- to late-stage infection.</p></div><div class=card-footer><span><a class="btn btn-outline-info ml-1 pl-2 mb-2" href target=_blank , rel=noopener role=button><i class="far fa-file-alt" style=color:#3b96cf></i>Publication</a></span></div></div></div></div></div></div></div></div><div class="container-fluid section-holder d-flex bg-white"><div class="container-fluid anchor pb-5 accomplishments-section" id=talks><h1 class=text-center>Recent & Upcoming Talks</h1><h6 class=text-center><em></em></h6><div class=container><div class=row id=acomplishment-card-holder><div class="col-md-12 col-lg-6 p-2"><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Statistical Methods for Analyzing Phage-ImmunoPrecipitation Sequencing Data</h5><div class=sub-title><span><a href="https://www.youtube.com/watch?v=0dNa5N3BEnI" title="Final Dissertation Defense" target=_blank rel=noopener>Final Dissertation Defense</a> | Feb 2022</span><br><span>Virtual</span></div></div><div class=card-body><p></p></div><div class=card-footer><span><a class="btn btn-outline-info ml-1 pl-2 mb-2" href target=_blank , rel=noopener role=button><i class="fa fa-video" style=color:#6493d1></i>Video</a></span></div></div></div><div class="col-md-12 col-lg-6 p-2"><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Antibody Profiling Identifies Antibody Targets Associated with Natural HIV Control</h5><div class=sub-title><span><a href=https://www.croiconference.org/croi-2021/ title="Conference on Retroviruses and Opportunistic Infections" target=_blank rel=noopener>Conference on Retroviruses and Opportunistic Infections</a> | Mar 2021</span><br><span>Virtual</span></div></div><div class=card-body><p></p></div><div class=card-footer><span><a class="btn btn-outline-info ml-1 pl-2 mb-2" href target=_blank , rel=noopener role=button><i class="fa fa-video" style=color:#6493d1></i>Video</a></span></div></div></div><div class="col-md-12 col-lg-6 p-2"><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Introduction to workflowr</h5><div class=sub-title><span><a href title="JHSPH Biostatistics Computing Club" target=_blank rel=noopener>JHSPH Biostatistics Computing Club</a> | Dec 15, 2020</span><br><span>Virtual</span></div></div><div class=card-body><p></p></div><div class=card-footer><span><a class="btn btn-outline-info ml-1 pl-2 mb-2" href target=_blank , rel=noopener role=button><i class="fab fa-github" style=color:#6493d1></i>Code</a></span></div></div></div><div class="col-md-12 col-lg-6 p-2"><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Introduction to JAGS and rJAGS</h5><div class=sub-title><span><a href title="JHSPH Biostatistics Computing Club" target=_blank rel=noopener>JHSPH Biostatistics Computing Club</a> | Apr 28, 2020</span><br><span>Virtual</span></div></div><div class=card-body><p></p></div><div class=card-footer><span><a class="btn btn-outline-info ml-1 pl-2 mb-2" href target=_blank , rel=noopener role=button><i class="fab fa-github" style=color:#6493d1></i>Code</a></span></div></div></div><div class="col-md-12 col-lg-6 p-2"><div class="card mt-3"><div class=card-header><h5 class="card-title mb-0">Top Scoring Pairs Classifier for Identifying Recent HIV Infection</h5><div class=sub-title><span><a href title="Johns Hopkins Biostatistics Seminar" target=_blank rel=noopener>Johns Hopkins Biostatistics Seminar</a> | Apr 6, 2020</span><br><span>Virtual</span></div></div><div class=card-body><p></p></div><div class=card-footer><span><a class="btn btn-outline-info ml-1 pl-2 mb-2" href target=_blank , rel=noopener role=button><i class="fas fa-tv" style=color:#6493d1></i>Slides</a></span></div></div></div></div></div></div></div><div class="container-fluid section-holder d-flex bg-dimmed"><div class="container-fluid anchor pb-5 education-section" id=teaching><h1 class=text-center>Teaching</h1><div class=container><table class=education-info-table style=background-color:#e6f5ff><tbody><tr><td class=icon><div class=hline></div><div class=icon-holder><i class="fas fa-university" style=color:#f9fafc></i></div></td><td class=line><div></div></td><td class=details><div class="degree-info card"><div class=row><div class="col-lg-10 col-md-8"><h5><a href=https://publichealth.jhu.edu/departments/biostatistics title="Johns Hopkins Bloomberg School of Public Health" target=_blank rel=noopener>Johns Hopkins Bloomberg School of Public Health</a></h5></div><div class="timeframe col-lg-2 col-md-4">2017-2022</div></div><h6>Graduate Teaching Assistant</h6><div class=extracurricular-activities><h6 class=text-muted>Responsibilities</h6><ul><li>Guided students through lab exercises and discussions of course material.</li><li>Provided feedback on assignments to facilitate a better understanding of course concepts.</li></ul></div><div class=taken-courses><table><thead><th>Term</th><th>Course Name</th><th>Instructor(s)</th></thead><tbody><tr class=course><td>Fall 2021</td><td>Statistical Computing</td><td>Stephanie Hicks, PhD</td></tr><tr class=course><td>Spring 2021</td><td>Bayesian Methods I and II</td><td>Gary Rosner, PhD and Robert Scharpf, PhD</td></tr><tr class=course><td>Fall 2019/2020</td><td>Advanced Data Science</td><td>Jeff Leek, PhD; Stephanie Hicks, PhD; and Roger Peng, PhD</td></tr><tr class=course><td>Summer 2020</td><td>Data Wrangling with R</td><td>Andrew Jaffe, PhD and John Muschelli, PhD</td></tr><tr class=course><td>Summer 2020</td><td>Introduction to R for Public Health Researchers</td><td>Andrew Jaffe, PhD and John Muschelli, PhD</td></tr><tr class=course><td>Spring 2020</td><td>Statistics for Laboratory Scientists I and II</td><td>Ingo Ruczinski, PhD</td></tr><tr class=course><td>Spring 2019</td><td>Statistical Methods in Public Health III and IV</td><td>Marie Diener-West, PhD; Leah Jager, PhD; and James Tonascia, PhD</td></tr><tr class=course><td>Fall 2018</td><td>Methods in Biostatistics I and II</td><td>Ciprian Crainiceanu, PhD</td></tr></tbody></table></div></div></td></tr><tr><td class=icon><div class=hline></div><div class=icon-holder><i class="fas fa-graduation-cap" style=color:#f9fafc></i></div></td><td class=line><div></div></td><td class=details><div class="degree-info card"><div class=row><div class="col-lg-10 col-md-8"><h5><a href=https://www.jhu.edu/ title="Johns Hopkins University" target=_blank rel=noopener>Johns Hopkins University</a></h5></div><div class="timeframe col-lg-2 col-md-4">2016-2017</div></div><h6>Undergraduate Teaching Assistant</h6><div class=extracurricular-activities><h6 class=text-muted>Responsibilities</h6><ul><li>Developed computer lab exercises on bioinformatic techniques to analyze omic data sets.</li><li>Assisted students with homework and computing lab assignments.</li><li>Led review sessions to reinforce topics introduced in lecture</li></ul></div><div class=taken-courses><table><thead><th>Term</th><th>Course Name</th><th>Instructor(s)</th></thead><tbody><tr class=course><td>Spring 2017</td><td>Introduction to Bioinformatics</td><td>Patrick Fleming, PhD</td></tr><tr class=course><td>Fall 2016</td><td>Biochemistry I</td><td>Patrick Fleming, PhD</td></tr><tr class=course><td>Summer 2016</td><td>Discrete Mathematics</td><td>Fred Torcaso, PhD</td></tr></tbody></table></div></div></td></tr></tbody></table></div></div></div><footer id=footer class="container-fluid text-center align-content-center footer pb-2"><div class="container pt-5"><div class="row text-left"><div class="col-md-4 col-sm-12"><h5>Contact me:</h5><ul><li><span>Address:</span> <span>615 N. Wolfe St., E3035, Baltimore, MD 21202</span></li><li><a href=mailto:achen70@jhu.edu target=_blank rel=noopener><span><i class="fas fa-envelope"></i></span><span>achen70@jhu.edu</span></a></li></ul></div></div></div><hr><div class=container><div class="row text-left"><div class=col-md-4><a id=theme href=https://github.com/hossainemruz/toha target=_blank rel=noopener><img src=/images/theme-logo_hu8376fd15465fef26ffe66b6bcf0ca686_13669_32x0_resize_box_2.png alt="Toha Theme Logo">
Toha</a></div><div class="col-md-4 text-center">© 2021 Copyright.</div><div class="col-md-4 text-right"><a id=hugo href=https://gohugo.io/ target=_blank rel=noopener>Powered by
<img src=/images/hugo-logo.svg alt="Hugo Logo" height=18></a></div></div></div></footer><script type=text/javascript src=/js/jquery-3.4.1.min.js></script><script type=text/javascript src=/js/popper.min.js></script><script type=text/javascript src=/js/bootstrap.min.js></script><script type=text/javascript src=/js/navbar.js></script><script type=text/javascript src=/js/plyr.js></script><script type=text/javascript src=/js/main.js></script><script src=/js/itype.min.js></script><script src=/js/github-button.js></script><script src=/js/home.js></script><script src=/js/jquery.filterizr.min.js></script></body></html>